public_html/events.php

```diff
@@ -343,14 +351,22 @@ function print_events($events, $is_past_event) {
 $events = [];
 $year_dirs = glob($md_base . 'events/*', GLOB_ONLYDIR);
 foreach ($year_dirs as $year) {
-    $event_mds = glob($year . '/*.md');
-    foreach ($event_mds as $event_md) {
+    // Single page events
+    $event_mds = glob($year . '/*');
+    foreach ($event_mds as $fpath) {
+        if (is_file($fpath) && substr($event_md, -3) == '.md') {
+            $event_md = $fpath;
+            $url = '/events/' . basename($year) . '/' . str_replace('.md', '', basename($event_md));
+        } elseif (is_dir($fpath) && file_exists($fpath . '/index.md')) {
+            $event_md = $fpath . '/index.md';
```

Comment:
The following would fix the incorrect relative links (but feels quite hacky)
```suggestion
            $href_url_prepend = basename($markdown_fn) . '/';
            $event_md = $fpath . '/index.md';
```

Reply:
Works for me 🤷 

Markdown is valid so the hacky fix is only for the hacky back end 😅 Hopefully fixable with website v2